html-parser/build.gradle.kts

```diff
@@ -10,10 +10,11 @@ java {
 
 dependencies {
     implementation(projects.dsl)
-    api(projects.baseFetcher)
+    api(projects.fetcher.baseFetcher)
     api(Deps.jsoup)
 
-    "jsExecutionImplementation"(project(path = ":browser-fetcher", configuration = "default"))
+    // TODO figure out what jsExecutionImplementation is for
+    "jsExecutionImplementation"(project(path = ":fetcher:browser-fetcher", configuration = "default"))
```

Comment:
I can't figure out what `jsExecutionImplementation` is for. I can't see any other reference to it in the project.

Reply:
The html-parser can handle js execution inside an html document before actually parsing.
The idea here was to make browser-fetcher (that can render Javascript) an optional dependency to html-parser, so people that need that functionality could just pull both dependencies in their projects and others who really. Only need html parsing without js execution can just use html-parser

https://docs.gradle.org/current/userguide/feature_variants.html#sec::consuming_feature_variants

https://github.com/skrapeit/skrape.it/blob/master/html-parser/src/main/kotlin/it/skrape/core/Parser.kt#L22-L27

Reply:
Oh neat, I didn't know about that feature!